views/forms/ShareConfigurationForm/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from 'react'
+import { useForm } from 'react-hook-form'
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Typography,
+} from '@mui/material'
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as yup from 'yup'
+
+import ControlledMultiSelect from '../ControlledMultiSelect'
+
+const schema = yup.object().shape({
+  readers: yup.array().when(({ isFixed }) => {
+    if (isFixed === true) yup.required('Owner cannot be removed')
+  }),
+})
+
+const ShareConfigurationForm = ({
+  onClose,
+  onSubmit,
+  open,
+  options,
+  title,
+  initialValues,
+}) => {
+  const { control, reset, handleSubmit } = useForm({
+    resolver: yupResolver(schema),
+  })
+
+  useEffect(() => {
+    reset({ readers: initialValues })
+  }, [initialValues])
```

Comment:
This feels like a workaround because the form default values cannot be updated. I think we can change the implementation to work with our conventional way of setting default values. Happy to pair on that if you want.

Reply:
sure we can take a look at it

Reply:
[3f060e8](https://github.com/AMP-SCZ/dpdash/pull/487/commits/3f060e8279d72eb848b54d8b7eb594892d7449a8)